src/osd/PG.cc

```diff
@@ -805,7 +805,6 @@ void PG::send_cluster_message(
   ConnectionRef con = osd->get_con_osd_cluster(
     target, get_osdmap_epoch());
   if (!con) {
-    m->put();
```

Comment:
for posterity, the regression was introduced by d4a71c3c5369c7a881822a6eb7624e3e80cc53de